### PR TITLE
Report duration analytics(Issue 112)

### PR DESCRIPTION
**Pull request by abhishek71994**
Please check as let me know if this is satisfactory.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -41,6 +41,3 @@ typings
 # JetBrains
 .idea
 .awcache
-
-# Package-lock
```
Comment (drimpact) [inline .gitignore]: Why was this changed?

Comment (abhishek71994) [reply .gitignore]: I dont know I did a git pull and these things changed what would you like me to do? change them back to the way it was before?

**Comment by drimpact** [comment]
thanks @abhishek71994
